### DISCUSSION TITLE
助手打开没有反应

### DISCUSSION THREAD

**Discussion by Rociky** [KIvoy/nikke-assistant#2] [Q&A]
大佬你好，双击之后没有反应，管理员权限运行及先开游戏再开助手均没有反应，applog发在这里了，能帮我看一下具体的问题吗
[app_log.txt](https://github.com/KIvoy/nikke-assistant/files/10915607/app_log.txt)

**Comment by KIvoy** [comment]
@Rociky 老哥， 我看了下log， 里面是之前我打包测试的内容， 没有新的， 说明没有加载成功。 我下个版本会加个更加底层的报错来看看问题， 但现在唯一的方法感觉只能1. 重新下载 2. 尝试把程序加到防火墙白名单。

**Comment by mengyuyou** [comment]
老哥，这是我的app_log，也是打不开界面，但是在任务管理器可以看到在运行，连着点几次还能再在任务管理器同时看到几个在运行的，只能先直接结束任务退出，大佬麻烦看一下啦
[Uploading app_log.txt…]()

**Comment by KIvoy** [comment]
@Rociky @mengyuyou 老哥可以下一下v1.1版本试一下， 那里增加了最新的报错以及多屏支持。 如果还是不行可以把app_error, app_log 和app_output都发上来看看， 应该能找到问题。

**Comment by Rociky** [comment]
@KIvoy 大佬，好像还是打不开..然后那三个文件什么内容都没有

**Reply by KIvoy** [reply]
老哥 你看下NIKKE_ASSISTANT.INI的设置是不是：

```
[app_settings]
log_to_file=True
pre_action_delay=0.5
post_action_delay=0.5

[ui_settings]
lang=zh

[game_settings]
lang=zh
```

**Comment by KIvoy** [comment]
@Rociky @mengyuyou 老哥们可以试一下1.3版本， 应该修复了这个问题。
